contracts/Vault.vy

```diff
@@ -1672,8 +1692,14 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
 
     # We have a loss to report, do it before the rest of the calculations
     if loss > 0:
+        pricePerShare: uint256 = self._shareValue(10 ** self.decimals)
         self._reportLoss(msg.sender, loss)
 
+        if self.strategies[msg.sender].checkLoss:
```

Comment:
I was wondering if a more full fledge healthcheck contract is needed to be deployed here to do this:

```
 if HealthCheck(healtcheckAddr).checkLoss(msg.sender):
      assert (HealthCheck(healtcheckAddr).healthCheck(msg.sender)) // sends vault addr + strat addr for global or special check logic

``` 



And we can turn the strategies on and off on that one, with no changes to struct on BaseStrategy, that way that Settings contract can have the healthcheck logic